src/main/java/it/gov/pagopa/pu/organization/service/BrokerEncryptionService.java

```diff
@@ -0,0 +1,48 @@
+package it.gov.pagopa.pu.organization.service;
```

Comment:
In order to avoid a package with too many classes, let's partitionate service package based on entities
```suggestion
package it.gov.pagopa.pu.organization.service.broker;
```

Reply:
done